test/Microsoft.AspNet.Mvc.FunctionalTests/TryValidateModelTest.cs

```diff
@@ -87,5 +81,37 @@ public class TryValidateModelTest
             var body = await response.Content.ReadAsStringAsync();
             Assert.Equal("{}", body);
         }
+
+        [Fact]
+        public async Task TryValidateModel_CollectionsModel_ReturnsErrorsForInvalidProperties()
```

Comment:
We already have a test `TryValidateModelAfterClearingValidationErrorInParameter` for non-collection type scenario. One of the unit tests has prefix scenario. Both `TryValidateModel` and `TryUpdateModel` cases have same code path. They both call same method. So isn't having tests with `TryValidateModel` enough? Do you see a scenario where `TryUpdateModel` would behave differently with respect to clearing `ModelStateDictionary`?